plugin/default-plugins/playground/frontend/components/CustomMetadataForm/controlTypes.js

```diff
@@ -6,6 +6,7 @@ const controlTypes = [
   'node',
   'avatar',
   'name',
+  'date',
```

Comment:
It is a real date, I just `toString` it because you can't display it in the UI, Chrome complains.